bindings/swig/conf.i

```diff
@@ -357,13 +357,12 @@ def ConfigParser__getfloat(self, section, option, raw=False):
 ConfigParser.getfloat = ConfigParser__getfloat
 del ConfigParser__getfloat
 
-ConfigParser._boolean_states = {'1': True, 'yes': True, 'true': True, 'on': True,
-                                '0': False, 'no': False, 'false': False, 'off': False}
 def ConfigParser__getboolean(self, section, option, raw=False):
     v = self.get(section, option, raw=raw)
-    if v.lower() not in self._boolean_states:
+    try:
+        return OptionBool(False).fromString(v)
```

Comment:
This needs to be case-insensitive. It doesn't look like this logic is case-insensitive.

Reply:
@Conan-Kudo II am using fromString() from OptionBool now. And fromString() is case insensitive. There is a link to the apropriate code:
https://github.com/rpm-software-management/libdnf/blob/8f1fedf8551b72d6bc24018f5980714b3a103aeb/libdnf/conf/OptionBool.cpp#L38

Reply:
I see then.